docs/docsite/rst/network/user_guide/network_best_practices_2.5.rst

```diff
@@ -1,61 +1,35 @@
-**************************************
+************************
+Ansible Network Examples
+************************
 
+This document describes some examples of using Ansible to manage your network infrastructure.
 
-Overview
-========
-
-This document explains the best practices for using Ansible 2.5 to manage your network infrastructure.
-
-
-Audience
---------
-
-* This example is intended for network or system administrators who want to understand how to use Ansible to manage network devices.
-
+.. contents::
+   :local:
 
 Prerequisites
--------------
+=============
 
 This example requires the following:
 
-* **Ansible 2.5** (or higher) installed. See :doc:`../../installation_guide/intro_installation` for more information.
+* **Ansible 2.5** (or higher) installed. See :ref:`intro_installation_guide` for more information.
 * One or more network devices that are compatible with Ansible.
-* Basic understanding of YAML :doc:`../../reference_appendices/YAMLSyntax`.
-* Basic understanding of Jinja2 Templates. See :doc:`../../user_guide/playbooks_templating` for more information.
+* Basic understanding of YAML :ref:`yaml_syntax`.
+* Basic understanding of Jinja2 templates. See :ref:`playbooks_templating` for more information.
 * Basic Linux command line use.
 * Basic knowledge of network switch & router configurations.
 
 
+Groups and variables in an inventory file
+=========================================
 
-Concepts
-========
-
-This section explains some fundamental concepts that you should understand when working with Ansible Networking.
-
-Structure
-----------
-
-The examples on this page use the following structure:
-
-.. code-block:: console
-
-   .
-   ├── facts-demo.yml
-   └── inventory
-
-
-Inventory, Connections, Credentials: Grouping Devices and Variables
--------------------------------------------------------------------
-
-An ``inventory`` file is an INI-like configuration file that defines the mapping of hosts into groups.
+An ``inventory`` file is a YAML or INI-like configuration file that defines the mapping of hosts into groups.
 
 In our example, the inventory file defines the groups ``eos``, ``ios``, ``vyos`` and a "group of groups" called ``switches``. Further details about subgroups and inventory files can be found in the :ref:`Ansible inventory Group documentation <subgroups>`.
 
-Because Ansible is a flexible tool, there are a number of ways to specify connection information and credentials. We recommend using the ``[my_group:vars]`` capability in your inventory file. Here's what it would look like if you specified your ssh passwords (encrypted with Ansible Vault) among your variables:
+Because Ansible is a flexible tool, there are a number of ways to specify connection information and credentials. We recommend using the ``[my_group:vars]`` capability in your inventory file. Here's what it would look like if you specified your SSH passwords (encrypted with Ansible Vault) among your variables:
```

Comment:
The mention of Ansible Vault here is awkward. It implies this example is what a file encrypted with Vault looks like. Instead, move it to after the example, or just wait for the Vault section altogether.

Reply:
I take that back, I misread what you meant. This is fine.